deploy-manage/users-roles/cluster-or-deployment-auth/manage-elastic-user-cloud.md

```diff
@@ -1,36 +1,41 @@
-# Reset the `elastic` user password [ech-password-reset]
-
-You might need to reset the password for the `elastic` superuser if you cannot authenticate with the `elastic` user ID and are effectively locked out from an Elasticsearch cluster or Kibana.
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-password-reset-elastic.html
+  - https://www.elastic.co/guide/en/cloud/current/ec-password-reset.html
+  - https://www.elastic.co/guide/en/cloud-heroku/current/ech-password-reset.html
+applies_to:
+  deployment:
+    ece:
+    ess:
+navigation_title: ECH and ECE
```

Comment:
```suggestion
navigation_title: Reset elastic user password on ECH and ECE
```

I know it's longer, but it's not a page about built-in users on ECH and ECE....

Reply:
Maybe we should put it in the TOC after the ECK page, or at the end of the built-in users section.